ui_testing/pages/order_page.py

```diff
@@ -131,13 +132,33 @@ def create_new_order(self, material_type='', article='', contact='', test_plans=
         self.base_selenium.LOGGER.info(' Order created with no : {} '.format(order_no))
         return self.get_suborder_data()
 
+    def create_order_with_test_unit(self, material_type='', article='', contact='', test_units=[''],
```

Comment:
create_new_order does this job. No?

Reply:
@0xIslamTaha yes, but this function I didn't create it , it already on the master branch so Do you want to delete it

Reply:
@dianaeldemerdash Can u refer me to this method in master one?

Reply:
@0xIslamTaha I returned it same as the master branch because I didn't use it